### PR TITLE
feat: Add possibility to hide details button in spatial filter

### PR DESCRIPTION
**Pull request by MV88**
also add a utility function for creating
a polygon from center and radius

## Description

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [x] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue

**What is the current behavior?**

#<issue>

**What is the new behavior?**

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/utils/CoordinatesUtils.js [hunk 1]
```diff
@@ -1011,7 +1011,9 @@ export const makeBboxFromOWS = (lcOWS, ucOWS) => {
 
     return [lc[0], lc[1], uc[0], uc[1]];
 };
-
+export const getPolygonFromCircle = (center, radius, units = "meters", steps = 100) => {
```
Comment (mbarto) [inline web/client/utils/CoordinatesUtils.js:1014]: This change is not releated to the PR title. Could you split into another PR or at least to a different commit with a meaningful message? I would also add a test for it.

**Review by mbarto — CHANGES_REQUESTED** [review]

File: web/client/utils/CoordinatesUtils.js (hunk 1 above)
Comment (MV88) [reply web/client/utils/CoordinatesUtils.js:1014]: you caught me :)

**Comment by MV88** [comment]
@mbarto updated pr by removing non related stuff

**Review by mbarto — APPROVED** [review]
